kubetest/extract.go

```diff
@@ -386,7 +386,10 @@ func (e extractStrategy) Extract(project, zone string) error {
 	case version:
 		var url string
 		release := e.option
-		if strings.Contains(release, "+") {
+		re := regexp.MustCompile(`(v\d+\.\d+\.\d+-gke.\d+)$`) // v1.8.0-gke.0
```

Comment:
this special case is a worrying precedent, though I'm not sure how to address it.